### PR TITLE
Plot complex spectogram

### PR DESCRIPTION
**Pull request by tluebeck**
### Which issue(s) are closed by this pull request?

Closes #

### Changes proposed in this pull request:

-
-
-

### CODE REVIEW

**Review by ahms5 — APPROVED** [review]

**Review by f-brinkmann — CHANGES_REQUESTED** [review]
Thanks for the effort. Found a typo and an error message that might not be needed

File: pyfar/plot/_utils.py [hunk 1]
```diff
@@ -457,6 +457,34 @@ def _assert_and_match_data_to_side(data, signal, side):
...
+        mask = frequencies <= 0
+    elif side == 'right':
+        mask = frequencies >= 0
+    else:
+        raise ValueError('Invalid `side` parameter, pass either `left` or '
+                         '`right`.')
+
+    # get corresponding data
+    frequencies = frequencies[mask]
+    spectrogram = spectrogram[mask, ...]
```
Comment (f-brinkmann) [inline pyfar/plot/_utils.py:474]: I think this needs to be changed for multi-dimensional signals:

If `signal.cdim == 1` this works, but if for example `signal.cdim == 2` the frequencies are in the third dimension. Is this related to pull requests for maintaining the cshape?

**Review by f-brinkmann — CHANGES_REQUESTED** [review]
I found one point, where I was not sure...

File: pyfar/plot/_utils.py (hunk 1 above)
Comment (tluebeck) [reply pyfar/plot/_utils.py]: The [..., mask, :] thing works for cdim > 1 :)

Comment (f-brinkmann) [reply pyfar/plot/_utils.py]: And should also work for cdim == 1 after merging #666, right?

Comment (tluebeck) [reply pyfar/plot/_utils.py]: yes, that should work

**Review by f-brinkmann — APPROVED** [review]

**Pull request merged**